workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/internal/selection/InMemorySelectionUtilities.kt

```diff
@@ -0,0 +1,9 @@
+package com.r3.corda.lib.tokens.workflows.internal.selection
+
+import com.r3.corda.lib.tokens.contracts.states.FungibleToken
+import net.corda.core.contracts.StateAndRef
+import net.corda.core.identity.Party
+
+fun issuerPredicate(issuer: Party): ((StateAndRef<FungibleToken>) -> Boolean) = {
```

Comment:
What's this for again?

Reply:
Just some leftover from when I tried our different things, will move